seed/java-sdk/java-inline-path-parameters/src/main/java/com/seed/api/resources/imdb/requests/GetMovieRequest.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * This file was auto-generated by Fern from our API Definition.
+ */
+package com.seed.api.resources.imdb.requests;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.seed.api.core.ObjectMappers;
+import java.util.HashMap;
+import java.util.Map;
+
+@JsonInclude(JsonInclude.Include.NON_ABSENT)
+@JsonDeserialize(builder = GetMovieRequest.Builder.class)
+public final class GetMovieRequest {
```

Comment:
Should we expect to see the path parameter as a property here?

Reply:
Removed this, it was carried over from the other PR